smart_open/transport.py

```diff
@@ -54,7 +66,10 @@ def register_transport(submodule):
 
     for scheme in schemes:
         assert scheme not in _REGISTRY
-        _REGISTRY[scheme] = submodule
+        if hasattr(submodule, "MISSING_DEPS") and getattr(submodule, "MISSING_DEPS") is True:
```

Comment:
Is this not sufficient?

```suggestion
        if getattr(submodule, "MISSING_DEPS") is True:
```

If there is no such attribute, getattr should return the default (None), right?

Reply:
It's not, because `getattr` raises `AttributeError` when you use it like that. However, I looked closer at the API after your comment and it turns out if you explicitly specify the default, it does not raise. Thanks for that tip!